uiplib/settings.py

```diff
@@ -10,17 +11,22 @@
 
 
 class ParseSettings:
+    """Argument Parser class."""
 
-    def __init__(self):
-        self.settings = self.get_settings_from_file()
-        self.settings.update(self.get_settings_from_cli())
+    def get_settings(self, args):
+        """Return the settings from parser."""
```

Comment:
Okay bro.. I knew this would be coming, making changes, as requested.
But I couldn't sacrifice the upcoming tests module, so I'll reform it to provide both the functionalitites.